.eslintrc

```diff
@@ -1,6 +1,23 @@
 {
-    "extends": "airbnb",
+    "extends": [
+      "airbnb",
+      "plugin:jasmine/recommended",
```

Comment:
👍 (we should note that we keep this in sync with the other repositories when tests are added there)

Reply:
Indeed.